test/cucumber/steps/ParkingSpaceReminderSteps.groovy

```diff
@@ -28,4 +28,4 @@ When(~'user "([^"]*)" asks a reminder where he parked his car'){ String user ->
 
 Then(~'there is no changes in the reservations of the parking spot'){ ->
```

Comment:
Teste escrito de forma errada, Then(~'^there is no changes in the reservations of the parking spot$')